cypress/integration/rancid_tomatillos_spec.js

```diff
@@ -1,12 +1,12 @@
 describe('Rancid Tomatillos', () => {
   beforeEach(() => {
     cy.fixture('mockData').then(( data ) => {
-      cy.intercept('https://rancid-tomatillos.herokuapp.com/api/v2/movies', {
+      cy.intercept('https://rancid-tomatillos.herokuapp.com/api/v2/movies/632618/videos', {
```

Comment:
I'm still not sure why it wasn't working to have both intercepts happen in the same code block, or why it's working now with the videos intercept happening first, but I'm glad it's working! Thanks so much for figuring this out.